diff --git a/lib/coreBundle.js b/lib/coreBundle.js
index 8ed8bb5de1b12c7c050191123b22a119298826bb..70730c3864123ca8582c7f775f42a6411e195803 100644
--- a/lib/coreBundle.js
+++ b/lib/coreBundle.js
@@ -9317,6 +9317,7 @@ async function launchProcess(options) {
     // process group, making it possible to kill child process tree with `.kill(-pid)` command.
     // @see https://nodejs.org/api/child_process.html#child_process_options_detached
     detached: process.platform !== "win32",
+    windowsHide: true,
     env: options.env,
     cwd: options.cwd,
     shell: options.shell,
